### ISSUE TITLE
Russian Localization

### ISSUE THREAD

**Issue by tapbots** [tapbots/calcbot-localization#25]
Use this to coordinate any work

**Comment by inikitin** [comment]
@tapbots In 'Power' category, 'Pferdestärke' translates to Russian as "metric horsepower", which is the unit that we already have. They are equal units, so I wonder whether 'Pferdestärke' should be ignored in Russian translation, or if it should be translated phonetically.

**Comment by tapbots** [comment]
I would say add Pferdestärke to the ignore list and just then translate Metric Horsepower as the two separate words "Metric" and "Horsepower"?  There's really no way to disable a unit for a particular locale and just can't think of a good way to do that without possibly screwing up a lot of other things.

**Comment by inikitin** [comment]
@tapbots for some reason, in Typography category all four of 'Pixel' units are still displayed in English, even though they were translated. 

Another issue is that (UK) units still have '(UK)' postfix, instead of what I wrote in place of '(Imperial)'. It might actually be better to leave it that way for the sake of brevity, but ideally I could specify 'UK' abbreviation in Russian, so those instances could be fully translated.

**Comment by tapbots** [comment]
Good catch, the modifier type units were all sorts of screwed up on my side of things. I hopefully have them all fixed, give it a look and let me know if there are any issues.

**Comment by inikitin** [comment]
@tapbots should we translate App/Localizable.string: <key>Watch</key>? If I understand correctly that is a product name reference just like "iPhone", so wouldn't need literal (or any) translation?

**Comment by tapbots** [comment]
I think Watch should be localized, since we aren't saying Apple Watch. Apple seems to do the same thing in the Watch App.
